internal/pkg/flink/components/table_view_test.go

```diff
@@ -114,3 +115,36 @@ func (s *TableViewTestSuite) TestTableShouldSetTitle() {
 
 	require.Equal(s.T(), expected, actual)
 }
+
+func (s *TableViewTestSuite) TestTableShortcutsShouldDisplayPlay() {
+	materializedStatementResults := getResultsExample()
+
+	actual := s.tableView.getTableShortcuts(materializedStatementResults, false)
+
+	cupaloy.SnapshotT(s.T(), actual)
+}
+
+func (s *TableViewTestSuite) TestTableShortcutsShouldDisplayPause() {
+	materializedStatementResults := getResultsExample()
+
+	actual := s.tableView.getTableShortcuts(materializedStatementResults, true)
+
+	cupaloy.SnapshotT(s.T(), actual)
+}
+
+func (s *TableViewTestSuite) TestTableShortcutsShouldDisplayChangelogMode() {
```

Comment:
I get that they're "trying to test" two different things, but this test is the same as `TestTableShortcutsShouldDisplayPlay`. We could maybe reduce them to one.

Reply:
yeah makes sense, I merged the 4 tests into 2!